src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -18,16 +25,19 @@ export type CardKebabMenuButtonDeps = {
     issueDetailsTextGenerator: IssueDetailsTextGenerator;
     detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
     navigatorUtils: NavigatorUtils;
-};
+} & IssueFilingButtonDeps;
 
 export interface CardKebabMenuButtonState {
     isContextMenuVisible: boolean;
+    showNeedsSettingsContent: boolean;
```

Comment:
NIT: to be consistent either both should say 'show*' or both should be 'showing*'

Reply:
I can ix this in the refactoring PR, right now we should be consistent with the file issue button